app/utils/Peer.client.ts

```diff
@@ -432,8 +432,8 @@ export default class Peer {
 						let trackPromises = response.tracks.map((track) => {
 							if (track.mid) {
 								return this.#resolveTrack(track.mid)
-							}
-						})
+							} else return undefined
```

Comment:
Is this a runtime behavior change?

Reply:
no it happens anyway, but this fixes a lint error which asks it to be explicit